index.js

```diff
@@ -1,4 +1,12 @@
-const subscribe = ({ next, error, complete }) => source => {
+const subscribe = (listener = {}) => source => {
```

Comment:
Maybe it was a bad idea to default to `{}` here as it will allow the destructuring to go through but the call to `next` to fail as `next` could be `undefined` and not a `function`.